Libraries/Text/TextInput/RCTBaseTextInputViewManager.m

```diff
@@ -67,6 +67,12 @@ @implementation RCTBaseTextInputViewManager
 RCT_EXPORT_VIEW_PROPERTY(textContentType, NSString)
 RCT_EXPORT_VIEW_PROPERTY(passwordRules, NSString)
 
+#if TARGET_OS_OSX // TODO(macOS GH#774)
+RCT_EXPORT_VIEW_PROPERTY(onAutoCorrectChange, RCTBubblingEventBlock);
+RCT_EXPORT_VIEW_PROPERTY(onSpellCheckChange, RCTBubblingEventBlock);
+RCT_EXPORT_VIEW_PROPERTY(onGrammarCheckChange, RCTBubblingEventBlock);
```

Comment:
Any reason these bubble?

Reply:
That's the eventType used by most other change event handlers.
What else would you recommend?

Reply:
Ah, that makes sense. Legitimately didn't realize *change events tend to bubble :D, been kinda guessing when I make my own events what they should be.